docs/tutorial-model-and-deploy.md

```diff
@@ -247,6 +247,21 @@ To verify that your deployment was actually successful open <http://localhost:99
 
 ### Tutorials for relations (available soon)
 
+#### connectTo interface
+By means of the interface name of a connectTo operation, it can be specified more precisely when it is to be executed.
+The default interface name is  http://opentosca.org/interfaces/connections.
```

Comment:
Shouldn't we use `https`?
`http://opentosca.org/interfaces/connections`

Reply:
In the container, the default interfaces are defined with `http`: https://github.com/OpenTOSCA/container/blob/master/org.opentosca.container.core/src/main/java/org/opentosca/container/core/convention/Interfaces.java